### PR TITLE
ensure crontab backup file has strict permssions

### PR DESCRIPTION
**Pull request by jsegitz**
see #171

### CODE REVIEW

File: src/crontab.c [hunk 1]
```diff
@@ -574,6 +575,9 @@ static int backup_crontab(const char *crontab_path) {
 		}
 	}
 	
+	// ensure backup file has strict permssions. Crontabs are not readable for
+	// other users and might contain sensitive information
```
Comment (t8m) [inline src/crontab.c:579]: Please do not use C++ style comments.

File: src/crontab.c [hunk 2]
```diff
@@ -574,6 +575,9 @@ static int backup_crontab(const char *crontab_path) {
...
+	old_umask = umask(0077)
```
Comment (t8m) [inline src/crontab.c:580]: Missing `;`

**Review by t8m — CHANGES_REQUESTED** [review]

File: src/crontab.c (hunk 2 above)
Comment (jsegitz) [reply src/crontab.c:580]: copy and paste never works :(

**Review by t8m — APPROVED** [review]

**Comment by jsegitz** [comment]
thank you for accepting this

**Pull request merged**